core/bind/core_bind.cpp

```diff
@@ -1318,6 +1326,9 @@ void _OS::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_low_processor_usage_mode_sleep_usec", "usec"), &_OS::set_low_processor_usage_mode_sleep_usec);
 	ClassDB::bind_method(D_METHOD("get_low_processor_usage_mode_sleep_usec"), &_OS::get_low_processor_usage_mode_sleep_usec);
 
+	ClassDB::bind_method(D_METHOD("set_update_vital_only", "enabled"), &_OS::set_update_vital_only);
+	ClassDB::bind_method(D_METHOD("is_update_vital_only"), &_OS::is_update_vital_only);
```

Comment:
This could be bound as a property too I guess.

Also I think `Engine` could be a better location for this, but I understand that it might have been put here as it's related to the "low processor usage mode" (which could likely be moved in `master` too - `OS` should be for talking to the OS, `Engine` for configuration of engine behavior).

Reply:
This would be easy enough to move to Engine. In fact I may well have started with it in Engine, but probably did move it into OS as all the low processor mode stuff was in there already. But even if we do decide to keep it here in 3.x to maintain backward compatibility, moving to Engine in 4.x could be a good plan for all this stuff.

Reply:
Yeah for `3.x` I think it can stay in `OS`, there's a ton of engine config specific stuff there which could only be moved to `Engine` in `master` with a compat breakage. So it's better to stick to the "old" usage here.

Reply:
I've now bound as a property too as requested.